config/functional-development-amo.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
It would be nice to add some comments at the top to mention how this requires a `HOSTNAME=functional.test` value and also how this is intended for the functional test suite which is hosted on addons-server